frontend/src/components/Login/Login.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+
+import PropTypes from "prop-types";
+
+import Card from "../../UI/Card";
+
+import classes from "./Login.module.css";
+
+export default function Login({ setIsLogged }) {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const emailChangeHandler = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const passwordChangeHandler = (event) => {
+    setPassword(event.target.value);
+  };
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+
+    if (email.trim().length === 0) {
```

Comment:
Don't make condition tha return nothing, you can chain conditions like : 
`if (email.trim().length === 0 && password.trim().length === 0 && !email.includes("@") { ...make something }`

Reply:
changed